docs/src/intro/use_cases.md

```diff
@@ -18,51 +18,50 @@ the floodplains, which will likely improve extreme streamflow predictions.
-many observations stations are available to provide real-time information, many rivers are not
+provide boundary conditions for a fast and detailed flood inundation model (SFINCS). The
+domain of the flood model covers the complete North and East Coast of Australia. Although
+many observation stations are available to provide real-time information, many rivers are not
 covered. For these locations, wflow\_sbm models are used to provide this real-time information.
 Additionally, these models are used to provide projections for potential future scenarios.
-Using the HydroMT library, all wflow\_sbm models were automatically build. The high level of
-flexibility in spatial and temporal resolution, and the physically based nature of the concept
-make wflow\_sbm a very suitable model for ungauged basins. Furthermore, the model is both
-detailled and computationally efficient enough in order to be coupled to the fast flood
+Using the HydroMT library, all wflow\_sbm models were automatically built. The high level of
+flexibility in spatial and temporal resolution, and the physics-based nature of the concept
+makes wflow\_sbm a very suitable model for ungauged basins. Furthermore, the model is
+sufficiently detailed and computationally efficient for a coupling with the fast flood
 inundation model SFINCS.
 
 ![fig_case_flifs](../images/case_flifs_1.png)
 
 The results of this Proof of Concept are very promising. Technically, we were able to very
 quickly setup the wflow\_sbm models, couple them to the flood inundation models (SFINCS) and
-run the models operationally under the Delft-FEWS platform. For two basins, model validation
-was caried out, by comparing the results of wflow\_sbm against the observations and the results
-of calibrated URBS models. This validation showed that the uncalibrated wflow\_sbm model
-results were already quite satisfying, especially given the complex nature of these basins,
-including several small and big reservoirs. We could also demonstrate the potential for further
-calibration by changing the KsatHorFrac parameter.
+run the models operationally under the Delft-FEWS platform. Model validation was carried
+out for two basins by comparing the results of wflow\_sbm against the observations and the
+results of calibrated URBS models. This validation showed that the uncalibrated wflow\_sbm
+model results were already quite satisfying, especially given the complex nature of these
+basins, including several small and big reservoirs. We could also demonstrate the potential
+for further calibration by changing the KsatHorFrac parameter.
 
 Reference: De Kleermaeker, S., Maguire, S., Druery, C., Morales, Y., Lijnse, T., Hegnauer, M.,
-2022 (in preperation). Developing a real-time data and modelling framework for operational
+2022 (in preperation). Developing a real-time data and modeling framework for operational
 flood inundation forecasting in Australia. Hydrology Water and Resources Symposium.
```

Comment:
This reference is now available (not in prep anymore), I would also stick to the original spelling of a reference (modelling).

```suggestion
2022. Developing a real-time data and modelling framework for operational flood inundation forecasting in Australia. In Hydrology & Water Resources Symposium 2022 (HWRS 2022): The Past, the Present, the Future. Engineers Australia. https://search.informit.org/doi/10.3316/informit.916755150845355
```

Reply:
Done